src/DB_Command.php

```diff
@@ -456,6 +456,177 @@ public function tables( $args, $assoc_args ) {
 		}
 	}
 
+	/**
+	 * Display the database name and size as ASCII table.
+	 *
+	 * Displays the Display the database name and size as ASCII table for `DB_NAME` specified in wp-config.php.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--size_format]
+	 * : Displays the database size as a number.
+	 * ---
+	 * default: b
+	 * options:
+	 *  - b (bytes)
+	 *  - kb (kilobytes)
+	 *  - mb (megabytes)
+	 *  ---
+	 *
+	 * [--tables]
+	 * : Display each table name and size as ASCII table instead of the database size.
+	 *
+	 * [--format]
+	 * : table, csv, json
+	 * ---
+	 * default: table
+	 * options:
+	 *   - table
+	 *   - csv
+	 *   - json
+	 * ---
+	 *
+	 * [--scope=<scope>]
+	 * : Can be all, global, ms_global, blog, or old tables. Defaults to all.
+	 *
+	 * [--network]
+	 * : List all the tables in a multisite install. Overrides --scope=<scope>.
+	 *
+	 * [--all-tables-with-prefix]
+	 * : List all tables that match the table prefix even if not registered on $wpdb. Overrides --network.
+	 *
+	 * [--all-tables]
+	 * : List all tables in the database, regardless of the prefix, and even if not registered on $wpdb. Overrides --all-tables-with-prefix.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     $ wp db size
+	 *     +-------------------+------+---------+
+	 *     | Name              | Size | Bytes   |
+	 *     +-------------------+------+---------+
+	 *     | wordpress_default | 6 MB | 5865472 |
+	 *     +-------------------+------+---------+
+	 *
+	 *     $ wp db size --tables
+	 *     +-----------------------+-------+-------+
+	 *     | Name                  | Size  | Bytes |
+	 *     +-----------------------+-------+-------+
+	 *     | wp_users              | 64 KB | 65536 |
+	 *     | wp_usermeta           | 48 KB | 49152 |
+	 *     | wp_posts              | 80 KB | 81920 |
+	 *     | wp_comments           | 96 KB | 98304 |
+	 *     | wp_links              | 32 KB | 32768 |
+	 *     | wp_options            | 32 KB | 32768 |
+	 *     | wp_postmeta           | 48 KB | 49152 |
+	 *     | wp_terms              | 48 KB | 49152 |
+	 *     | wp_term_taxonomy      | 48 KB | 49152 |
+	 *     | wp_term_relationships | 32 KB | 32768 |
+	 *     | wp_termmeta           | 48 KB | 49152 |
+	 *     | wp_commentmeta        | 48 KB | 49152 |
+	 *     +-----------------------+-------+-------+
+	 *
+	 * 	   $ wp db size --size_format=b
+	 * 	   5865472
+	 *
+	 * 	   $ wp db size --size_format=kb
+	 * 	   5728
+	 *
+	 * 	   $ wp db size --size_format=mb
+	 * 	   6
+	 */
+	public function size( $args, $assoc_args ) {
+
```

Comment:
If I provide `--scope=<scope>`, `--network`, `--all-tables-with-prefix` or `--all-tables` without the `--tables` flag, what should happen?